src/utils.js

```diff
@@ -25,13 +25,16 @@ export const parseInclude = request => {
   const models = getModels(request);
   if (models.isBoom) return models;
 
-  return include.map(a => {
-    const singluarOrPluralMatch = Object.keys(models).find((modelName) => {
-      const { _singular, _plural } = models[modelName];
-      return _singular === a || _plural === a;
-    });
-
-    if (singluarOrPluralMatch) return models[singluarOrPluralMatch];
+  return include.map(b => {
+    const a = /^{.*}$/.test(b) ? JSON.parse(b) : b;
```

Comment:
https://github.com/mdibaiee/hapi-sequelize-crud/blob/5ba9d7d26100f1a1a82367097342a8c35d43a26f/src/utils.js#L50-L54 and https://github.com/mdibaiee/hapi-sequelize-crud/blob/5ba9d7d26100f1a1a82367097342a8c35d43a26f/src/utils.js#L78-L83 are both options. Thanks!